tests/methods/run_forte_tests_travis.py

```diff
@@ -165,4 +165,10 @@ class bcolors:
     failed_log.write("# %s\n" % now)
     failed_log.write("\n".join(failed))
     failed_log.close()
+
```

Comment:
Just FYI, I removed the printing of failed test cases (lines 128-132) added before this commit. So we only have one such printings.